distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -519,6 +518,127 @@ type Decimal
     / : Decimal -> Decimal
     / self (that : Decimal) = self.divide that
 
+    ## ALIAS modulo, modulus
+       GROUP Operators
+       ICON math
+       Computes the remainder when dividing this by that.
+
+       Arguments:
+       - that: The number to divide this by.
+
+       Remainder in Enso will undergo automatic conversions such that you need
+       not convert other numeric types to `Decimal` manually.
```

Comment:
This sounds to me like the _result_ of the operation will undergo conversions, which as far as I understand is not the case and you mean the _arguments_, right?

Maybe we can rephrase it to make it clearer?
```suggestion
       Arguments of the `remainder` operation will undergo automatic conversions such that you need
       not convert other numeric types to `Decimal` manually.
```

maybe something like that?
1. I don't think you need to say `in Enso` there - this is an Enso library so that is rather assumed,
2. IMO it's good to point you mean _arguments_

Reply:
Done.